db/migrate/20180830104245_add_creator_id_to_document.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddCreatorIdToDocument < ActiveRecord::Migration[5.2]
+  def change
+    add_column :documents, :creator_id, :integer
```

Comment:
Can we add a foreign key to the users table here, so that we won't be able to accidentally delete users that have documents attached?